master/buildbot/test/unit/test_db_logs.py

```diff
@@ -573,6 +573,80 @@ def test_deleteOldLogChunks_basic(self):
             lines = yield self.db.logs.getLogLines(logid, 0, logdict['num_lines'])
             self.assertEqual(lines, '')
 
+    @defer.inlineCallbacks
+    def test_deleteOldLogChunks_exceptions(self):
+        yield self.insertTestData(
+            self.backgroundData + [
```

Comment:
Would it be possible to define a separate `self.deleteBuildLogsData` and with full data set (i.e. inline the `self.backgroundData` there)? This way it would be easier to understand what data test operates on as it would all be in a single place.

Reply:
Got it.